### PR TITLE
Cleanup to support 1.20.2

### PR DESCRIPTION
**Pull request by AlexProgrammerDE**

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -0,0 +1,249 @@
+#!/bin/sh
```
Comment (rom1504) [inline gradlew:1]: are you sure this should be committed ?

File: gradlew.bat [hunk 2]
```diff
@@ -0,0 +1,92 @@
+@rem
```
Comment (rom1504) [inline gradlew.bat:1]: are you sure this should be committed ?

**Comment by rom1504** [comment]
looks ok; left a few comments

File: gradlew (hunk 1 above)
Comment (AlexProgrammerDE) [reply gradlew:1]: Yeah, the official gradle wrappers are commonly stored inside the repository. Maven also has wrappers, but those are used more rarely.

File: gradlew.bat (hunk 2 above)
Comment (AlexProgrammerDE) [reply gradlew.bat:1]: See my other reply: https://github.com/PrismarineJS/McDataExtracting/pull/13#discussion_r1429526030

File: gradlew (hunk 1 above)
Comment (AlexProgrammerDE) [reply gradlew:1]: https://stackoverflow.com/a/20351077

**Pull request merged**